azkaban-common/src/main/java/azkaban/executor/container/VPAFlowCriteria.java

```diff
@@ -0,0 +1,127 @@
+package azkaban.executor.container;
+
```

Comment:
Clarification: if we update the (project, flow) pair, we update the config and then restart the web server to be able to pick that. Is it correct to say? 

This change has a lot in common with a previous change https://github.com/azkaban/azkaban/pull/3152. Is there a possibility to re-use some parts of it for this use case?

Reply:
Good point. Addressed.